app/src/main/java/edu/uc/forbesne/shoppingsidekick/service/ProductService.kt

```diff
@@ -18,11 +18,11 @@ import retrofit2.Response
 
 class ProductService {
 
-    fun fetchProductsByName(productName: String) : MutableLiveData<ArrayList<Product>> {
+    internal fun fetchProductsByDescription(productDescription: String) : MutableLiveData<ArrayList<Product>> {
```

Comment:
Changed visibility modifiers from the best practice here:  https://www.baeldung.com/kotlin/visibility-modifiers